src/lib/seam/components/DeviceDetails/ThermostatDeviceDetails.tsx

```diff
@@ -95,10 +95,9 @@ export function ThermostatDeviceDetails(props: {
               tooltipContent={t.defaultSettingsTooltip}
             >
               <DetailRow label={t.defaultClimate}>
```

Comment:
Should this be changed?
```suggestion
              <DetailRow label={t.currentClimate}>
```